src/tags/sayt/autocomplete.ts

```diff
@@ -23,16 +23,22 @@ export function reset() {
   selectedNode = searchInput;
 }
 
-function swap<T extends Element>(selected: T, next: T): T {
+function swap<T extends Element>(autocompleteList: Element, selected: T, next: T): T {
   if (next) {
-    selected.classList.remove(ACTIVE);
+    removeActive(autocompleteList);
     next.classList.add(ACTIVE);
     if (next.firstElementChild) queryUpdateNotifier(next.getAttribute(DATA_VALUE));
     return next;
   }
   return selected;
 }
 
+function removeActive(autocompleteList: Element) {
+  let test = autocompleteList.getElementsByClassName("gb-autocomplete__item");
+  var array = Array.from(test);
```

Comment:
instead of this do `Array.from(test).forEach(element => element.classList.remove('active'))`
note the single quotes